### ISSUE TITLE
Question: forceready ur whole team

### ISSUE THREAD

**Issue by knoeppi** [splewis/csgo-pug-setup#358]
Hey, 

we were using Get5 before on our old server but switched to a new one with pugsetup and practicemode.
Is there a way to have the forceready in pugsetup work like it does in Get5 so it ready ups your whole team? I thought about copying the code from Get5 over to pugsetup but Im not sure if I will destroy anything with that. 

Any help would be greatly appreciated

**Comment by MoritzLoewenstein** [comment]
There is a command for this: `sm_forcestart`.
You should be able to call it with `!forcestart` (you can also add aliases in `chataliases.cfg`.

**Comment by knoeppi** [comment]
Yeah but that way it doesnt matter if the enemy team is ready or?
I dont want to start the game instantly just set the state of me and my team m8s on ready without everybody typing ".ready" (or like that")

**Comment by MoritzLoewenstein** [comment]
Thats not possible currently.

**Comment by knoeppi** [comment]
Okay ty.

And if I copy the function of forceready from Get5 or better not and just leave it as it is!?

**Comment by splewis** [comment]
This could be added, but I don't know how easy it is. It's probably pretty trivial - PRs welcome.

As a workaround, you could just `.forceready` the players independently, if you have the admin powers to that command (that's what I used to do myself) .
